src/d_deh.c

```diff
@@ -2229,10 +2229,10 @@ void deh_procSounds(DEHFILE *fpin, FILE* fpout, char *line)
                   S_sfx[indexnum].volume = value;
                 else
                   if (!strcasecmp(key,deh_sfxinfo[6]))  // Zero 4
-                    S_sfx[indexnum].data = (void *)(intptr_t) value; // killough 5/3/98: changed cast
+                    ; // ignored
                   else
                     if (!strcasecmp(key,deh_sfxinfo[7]))  // Neg. One 1
-                      S_sfx[indexnum].usefulness = value;
+                      ; // ignored
```

Comment:
We should probably also ignore the next field which sets `lumpnum`. I mean, how certain do we want to crash?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/971/commits/6dd78d548cccf7eeef441256079f7a5a00c0e2b4